static/js/addons/cl.utils.js

```diff
@@ -48,16 +64,40 @@ var Cl = window.Cl || {};
             }
         },
 
-        // INFO: simple mobile and tablet filter that can be extended
-        mobile: function () {
-            return (navigator.userAgent.match(/iPhone/i) || navigator.userAgent.match(/Android/i));
+        /**
+         * Simple redirection
+         *
+         * @method redirectTo
+         * @param url {String} URL string
+         */
+        redirectTo: function (url) {
+            window.location.href = url;
         },
-        tablet: function () {
-            return (navigator.userAgent.match(/iPad/i));
-        }
 
-        // INFO: add here more Utilities required for the project
+        /**
+         * Save information within local storage
+         *
+         * @method setStorage
+         * @param token {String} namespace
+         * @param value {String} storage value
+         */
+        setStorage: function (token, value) {
+            if (window.localStorage) {
```

Comment:
Error here
http://stackoverflow.com/questions/11214404/how-to-detect-if-browser-supports-html5-local-storage